mono/metadata/metadata.h

```diff
@@ -309,6 +309,12 @@ typedef struct {
 	unsigned int token    : 31;
 } MonoCustomMod;
 
+typedef struct _MonoCustomModContainer {
+	uint8_t count; /* max 64 modifiers follow at the end */
+	void *image; /* Image containing types in modifiers array */
```

Comment:
Also does `MonoCustomModContainer` really need to be in the public API already?

Reply:
Custom modifiers are used by monodis. I'm not sure how the public API consumers would be able to interact with the mods without the image, count, and mods provided here.